packages/paste-website/src/pages/components/badge/index.mdx

```diff
@@ -91,55 +99,98 @@ export const pageQuery = graphql`
+    Decorative 20
+  </Badge>
+  <Badge as="span" variant="decorative30">
+    Decorative 30
+  </Badge>
+  <Badge as="span" variant="decorative40">
+    Decorative 40
+  </Badge>
+  <Badge as="span" variant="neutral_counter">
+    1
+  </Badge>
+  <Badge as="span" variant="error_counter">
+    1
+  </Badge>
+</Box>
+`.trim()}
+</LivePreview>
+
 ## Guidelines
 
-### About Badge
+## About Badge
+
+A Badge can be used to label a piece of UI for quick identification.
+It can be used in a wide variety of situations, such as labeling Beta product features
+or an attribute such as “Inbound” or “Outbound”. A Badge can also be used to show a count,
+such as the number of new messages.
+
+A Badge should contain text content and can optionally contain an Icon.
+Text length should be kept short.
 
-A Badge can be used to label a piece of UI for quick identification and navigation. It can be used in a wide variety of situations such as labelling Beta product features or a state, for example a status.
+### Accessibility
 
-A Badge should contain text content and can optionally contain smaller graphical elements such as Icons and Avatars. Text length should be kept short.
+- Ensure that Badge text is clear enough to understand immediately without having to rely on color to communicate meaning.
+  For example, an error badge should say “Token missing” rather than just “Token”, so the error is communicated via the Badge label.
+
+- For Status Badges, the inclusion of an icon is recommended to further reinforce the meaning.
 
 ## Examples
 
-### Default Badge
+### Status
 
-<LivePreview scope={{Box, Badge, InformationIcon, ErrorIcon, WarningIcon, SuccessIcon, NewIcon}} language="jsx">
-  {`<Box display="flex" columnGap="space80">
-  <Badge as="span" variant="default">
-    Default
-  </Badge>
-  <Badge as="span" variant="default">
-    <InformationIcon decorative />
-    Default
-  </Badge>
-  <Badge as="span" variant="default">
-    Default
-    <InformationIcon decorative />
-  </Badge>
-</Box>
-`.trim()}
-</LivePreview>
+Status Badges should be used in cases where the Badge is intended to communicate a specific,
+semantic attribute, such as “warning” or “new”. Status Badges should generally have an Icon that
+should be prefixed before the text.
+
+#### Neutral
 
-### Info Badge
+Use the Neutral badge to highlight neutral attributes of an object. When using the Neutral Badge,
+the inclusion of an Information icon is optional.
 
 <LivePreview scope={{Box, Badge, InformationIcon, ErrorIcon, WarningIcon, SuccessIcon, NewIcon}} language="jsx">
   {`<Box display="flex" columnGap="space80">
-  <Badge as="span" variant="info">
-    Info
+  <Badge as="span" variant="neutral">
+    Neutral
   </Badge>
-  <Badge as="span" variant="info">
+  <Badge as="span" variant="neutral">
     <InformationIcon decorative />
-    Info
+    Neutral
   </Badge>
-  <Badge as="span" variant="info">
-    Info
+  <Badge as="span" variant="neutral">
```

Comment:
Can we get rid of all the examples that have the icon after the text, since we're recommending that the icon go before the text?